pkg/hubble/parser/sock/parser_test.go

```diff
@@ -398,15 +394,60 @@ func TestDecodeSockEvent(t *testing.T) {
 				Summary:        "TCP",
 			},
 		},
+		{
+			name: "custom decoder",
```

Comment:
This test uses a custom deccoder, but it doesnt' look like it does anything special, is that right? I guess it sets the flow UUID?